src/css/custom.css

```diff
@@ -68,3 +68,14 @@
   border-radius: var(--ifm-code-border-radius);
   padding: 2em 1em 1em;
 }
+
+.text-center {
+  text-align: center;
+}
```

Comment:
don't we have it already somewhare here?
https://docusaurus.io/docs/styling-layout#styling-your-site-with-infima

Reply:
Unfortunately, I couldn't find something similar. Infima doesn't seem to have as many utility classes.

Reply:
Instead, it exists, it's not written in the documentation, but by trying, using Infima's style 'with double dash' I managed to use Infima's utility class. I will remove this line of code.